src/components/SearchResults.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "../styles/search-results.css";
+
+const SearchResults = ({ searched, results }) => {
```

Comment:
Thinking about it, do you need the `searched` prop passing down into the component? If your just using it to determine if the component should be rendered, why not just use conditional logic in the App component, eg:
```
{searched && <SearchResults results={results} />}
```
This would mean you could cut some unnecessary logic from your tests for `SearchResults` too